src/ducks/Watchlists/Actions/Edit/EditForm/index.js

```diff
@@ -48,26 +51,42 @@ const EditForm = ({
       name === defaultSettings.name &&
       description === defaultSettings.description &&
       isPublic === defaultSettings.isPublic &&
+      listItems === defaultSettings.listItems &&
       id
     ) {
       toggleOpen(false)
     } else {
-      onFormSubmit({ name, description, isPublic })
+      onFormSubmit({ name, description, isPublic, listItems })
     }
   }
 
+  function checkIsTouched (key, value) {
+    const _value =
+      value === '' || value === undefined || value === null ? null : value
```

Comment:
Can be simplified to `const normalizedValue = value || null`. If `value` can be a number, then `value === 0 ? 0 : (value || null)`